src/Configuration/Config.Json/src/JsonConfigurationFileParser.cs

```diff
@@ -24,8 +24,14 @@ internal class JsonConfigurationFileParser
         {
             _data.Clear();
 
+            var jsonReaderOptions = new JsonReaderOptions
```

Comment:
We probably need to expose this on the AddJson level if we are going to stray from the defaults